pkg/apim/APIMUtility.go

```diff
@@ -51,3 +66,21 @@ func (apim *Bundle) IsPayloadJSON() bool {
 	var js json.RawMessage
 	return json.Unmarshal([]byte(apim.Payload), &js) == nil
 }
+
+func (odataFilters *OdataParameters) InitOdataQuery() (string, error) {
+
+	v := reflect.ValueOf(odataFilters).Elem()
```

Comment:
Maybe with a better name for this variable (instead of `v`, or `value`) the code can become more understandable.

Reply:
fixed